db/migrate/20190527191030_create_device_table.rb

```diff
@@ -0,0 +1,10 @@
+class CreateDeviceTable < ActiveRecord::Migration[5.2]
+  def change
+    create_table :devices do |t|
+      t.string   :name
+      t.string   :device_type
```

Comment:
didn't see device type in the form fields.. as expected?

Reply:
for me its hard to say, maybe it will be the next task to work wit it

Reply:
ok i will delete all related with devices according to Igor comment